README.md

```diff
@@ -210,7 +210,7 @@ pip install -r requirements.lock
 # Generate protos
 python -m grpc_tools.protoc --proto_path=. \
   --python_out=. --grpc_python_out=. --pyi_out=. \
-  protos/grpc/testing/*.proto
+  protos/grpc/testing/*.proto protos/grpc/testing/**/*.proto 
```

Comment:
we need to add this to https://github.com/grpc/psm-interop/blob/657acc2637b34ea167c5289762f02be07dae7042/.github/workflows/psm-interop.yaml#L49-L55